src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import './utils/variable';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-calendar;
+  gap: $day-gap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    height: $cell-hight;
```

Comment:
There is a typo in the variable name `$cell-hight`. It should be `$cell-height`. Ensure that the variable is correctly defined in your variables file.